docs/source/en/quantization.md

```diff
@@ -26,6 +26,61 @@ Interested in adding a new quantization method to Transformers? Read the [HfQuan
 
 </Tip>
 
+## Quanto
+
+[🤗 Quanto](https://github.com/huggingface/quanto) library is a versatile pytorch quantization toolkit. The quantization method used is the linear quantization. Quanto provides several unique features such as:
```

Comment:
TODO (in the future) add the link to the colab notebook + mention that quanto is modality agnostic (most users will tend to think that this is only for LLMs)